cmd/snap-confine/mount-support-nvidia.c

```diff
@@ -345,9 +345,13 @@ static void sc_mount_nvidia_driver_biarch(const char *rootfs_dir)
 
 #ifdef NVIDIA_MULTIARCH
 
+#define SC_NVIDIA_VERSION_MAX_LEN 42
+
 struct sc_nvidia_driver {
 	int major_version;
 	int minor_version;
+	int micro_version;
+	char version_string[SC_NVIDIA_VERSION_MAX_LEN+1];
```

Comment:
I don't think we need to lug the original version string around. There's no real use of it in this PR.